aiohue/v2/models/feature.py

```diff
@@ -296,10 +296,13 @@ class EffectStatus(Enum):
     CANDLE = "candle"
     FIRE = "fire"
     SPARKLE = "sparkle"
+    GLISTEN = "glisten"
+    OPAL = "opal"
+    PRISM = "prism"
```

Comment:
Do you think these additions should be part of this PR?

Reply:
Nah, but it was too much work to create a new PR just for these ;-)
I spotted these by accident